core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -391,7 +298,7 @@ class ConfigCommandTest extends QuorumTestHarness with Logging {
   def shouldFailIfUnrecognisedEntityTypeUsingZookeeper(): Unit = {
     val createOpts = new ConfigCommandOptions(Array("--zookeeper", zkConnect,
       "--entity-name", "client", "--entity-type", "not-recognised", "--alter", "--add-config", "a=b,c=d"))
-    assertThrows(classOf[IllegalArgumentException], () => ConfigCommand.alterConfigWithZk(null, createOpts, new DummyAdminZkClient(zkClient)))
+    assertThrows(classOf[IllegalArgumentException], () => ConfigCommand.alterConfigWithZk(null, createOpts, new DummyAdminZkClient(null)))
```

Comment:
nit: we can create a DummyAdminZKClient with null zkClient, to use in this new unit test.

Reply:
Done